### PR TITLE
Added notes for Jupyter & Colab users

### PR DESCRIPTION
**Pull request by sshiv5768**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,9 @@ has been under active development since then.
 For usage questions, the best place to go to is [StackOverflow](https://stackoverflow.com/questions/tagged/pandas).
 Further, general questions and discussions can also take place on the [pydata mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata).
 
+## Notes
+If you are using Jupyter notebook or Google's Colaboratory then pandas are pre-installed in it.
```
Comment (rhshadrach) [inline README.md:157]: Is this true for Jupyter? I believe it's possible to have Jupyter installed without pandas, at the very least via `conda remove --force pandas`.

**Review by rhshadrach — COMMENTED** [review]
What is your motivation for adding this to the README?

File: README.md (hunk 1 above)
Comment (MarcoGorelli) [reply README.md:157]: or if you install Jupyter without going through conda. Thanks for the suggestion, but IMO this doesn't need to be in the README (though perhaps a note on google colab would go well somewhere else in the docs)

**Comment by TomAugspurger** [comment]
I think we'll pass on this. It doesn't need to be mentioned there.

**Pull request closed without merging**